### ISSUE TITLE
Vebus: Bugs in current sensor assistant detection after Multi reset

### ISSUE THREAD

**Issue by vejpasop** [victronenergy/venus#763]
Customer states that after reset of his three system Multi's with three AC current sensors not always all current sensors are discovered. 

Rebooting the CCGX fixes the issue,

There are actual two issues:

1.  Not all current sensors are always discovered
2.  ```dbus_vebus_to_pvinverter``` crashes as soon as it tries to to export a second Dbus service for ac sensors on a different location (_input1, _output or _input2).

(Detailed issue descriptions can be found in the comments below)

- [x] Issue 1 can be fixed in mk2-dbus. ```dbus_vebus_to_pvinverter```  monitors the AC sensor count item in order to update the AC sensor information. If at the time the count value is set the sensor position and phase items are not yet initialized the non initialized sensors will never be detected. The mk2-dbus fix makes sure that the count item is only updated after the position and phase items for all connected sensors are published. (Fix is pushed to master branch of mk2-dbus)

- [x] Issue 2 should be fixed in '``dbus_vebus_to_pvinverter```

**Comment by vejpasop** [comment]
Issue can be reproduced. 
The problem seems to be in dbus_vebus_to_pvinverter process.

System configuration:
- 2 Multis in parallel. 
- Master L1 contains AC current sensor at location 1
- Slave L1 contains AC current senor at location 1

run dbus_vebus_to_pvinverter

Reset Multi through ```com.victronenergy.vebus.ttyO1/SystemReset```. After a couple of resets the problem will show.

**Normal  dbus_vebus_to_pvinverter.py output:** 
```
INFO:root:New service added, or sensorcount was changed on an existing service. Count for com.victronenergy.vebus.ttyO1 is now 2
INFO:root:AC Sensor on /AcSensor/0, location: 1, phase: L1
INFO:root:AC Sensor on /AcSensor/1, location: 1, phase: L1
INFO:root:registered ourselves on D-Bus as com.victronenergy.pvinverter.vebusacsensor_output
INFO:root:Added to D-Bus: PV Inverter on output containing 2 AC-sensors on L1, 0 AC-sensors on L2, 0 AC-sensors on L3
```

**Output when things go wrong:**
```
INFO:root:New service added, or sensorcount was changed on an existing service. Count for com.victronenergy.vebus.ttyO1 is now 2
INFO:root:AC Sensor on /AcSensor/0, location: 1, phase: L1
INFO:root:registered ourselves on D-Bus as com.victronenergy.pvinverter.vebusacsensor_output
INFO:root:Added to D-Bus: PV Inverter on output containing 1 AC-sensors on L1, 0 AC-sensors on L2, 0 AC-sensors on L3
```

It does report a count of 2. dbus-monitor output show a proper properties changed for both vebus sensors path
```

signal time=1610096491.427342 sender=:1.6112 -> destination=(null destination) serial=8864 path=/AcSensor/0/Location; interface=com.victronenergy.BusItem; member=PropertiesChanged
   array [
      dict entry(
         string "Value"
         variant             uint32 1
      )
      dict entry(
         string "Text"
         variant             string "1"
      )
   ]
method call time=1610096491.534154 sender=:1.6128 -> destination=:1.6112 serial=1564 path=/AcSensor/0/Location; interface=(null); member=GetValue
method call time=1610096491.686276 sender=:1.6128 -> destination=:1.6112 serial=1588 path=/AcSensor/1/Location; interface=(null); member=GetValue
signal time=1610096491.787517 sender=:1.6112 -> destination=(null destination) serial=8877 path=/AcSensor/1/Location; interface=com.victronenergy.BusItem; member=PropertiesChanged
   array [
      dict entry(
         string "Value"
         variant             uint32 1
      )
      dict entry(
         string "Text"
         variant             string "1"
      )
   ]
```
The ```dbus_vebus_to_pvinverter``` process does however not get the values for the second senor

**Comment by vejpasop** [comment]
While testing with a different configuration an other bug in ```dbus_vebus_to_pvinverter``` showed.

System configuration:
-  2 Multis in parallel.
-  Master L1 contains AC current sensor at location 1
-  Slave L1 contains AC current senor at location **0** (<-- Different location)

With this configuration ```dbus_vebus_to_pvinverter``` crashes and  restart all the time.

```
INFO:root:Found: com.victronenergy.vebus.ttyO1, checking for valid AC Current Sensors
INFO:root:New service added, or sensorcount was changed on an existing service. Count for com.victronenergy.vebus.ttyO1 is now None
INFO:root:New service added, or sensorcount was changed on an existing service. Count for com.victronenergy.vebus.ttyO1 is now 2
INFO:root:AC Sensor on /AcSensor/0, location: 0, phase: L1
INFO:root:AC Sensor on /AcSensor/1, location: 1, phase: L1
INFO:root:registered ourselves on D-Bus as com.victronenergy.pvinverter.vebusacsensor_input1
INFO:root:Added to D-Bus: PV Inverter on input 1 containing 1 AC-sensors on L1, 0 AC-sensors on L2, 0 AC-sensors on L3
Traceback (most recent call last):
  File "././ext/velib_python/vedbus.py", line 313, in _properties_changed_handler
    self._eventCallback(self._serviceName, self._path, changes)
  File "./dbus_vebus_to_pvinverter.py", line 267, in countchanged
    b.update_dbus_service()
  File "./dbus_vebus_to_pvinverter.py", line 144, in update_dbus_service
    self._dbusService = VeDbusService('com.victronenergy.pvinverter.vebusacsensor_' + pf[self._name])
  File "././ext/velib_python/vedbus.py", line 78, in __init__
    self._dbusnodes['/'] = self._create_tree_export(self._dbusconn, '/', self._get_tree_dict)
  File "././ext/velib_python/vedbus.py", line 145, in _create_tree_export
    return VeDbusTreeExport(bus, objectPath, get_value_handler)
  File "././ext/velib_python/vedbus.py", line 321, in __init__
    dbus.service.Object.__init__(self, bus, objectPath)
  File "/usr/lib/python2.7/site-packages/dbus/service.py", line 485, in __init__
    self.add_to_connection(conn, object_path)
  File "/usr/lib/python2.7/site-packages/dbus/service.py", line 576, in add_to_connection
    self._fallback)
KeyError: "Can't register the object-path handler for '/': there is already a handler"
```

**Comment by vejpasop** [comment]
Issue 1: Fixed in mk2-dbus v3.17
Issue 2: Fixed in dbus_vebus_to_pvinverter v1.29
